code/game/objects/items/devices/laserpointer.dm

```diff
@@ -137,8 +137,7 @@
 
 	//catpeople
 	for(var/mob/living/carbon/human/H in view(1,targloc))
-		//if(!isfelinid(H) || H.incapacitated() || H.is_blind())
-		if(!(isfelinid(H) || HAS_TRAIT(H, TRAIT_FELINE)) || H.incapacitated() || H.is_blind()) //SKYRAT EDIT - FELINE TRAIT
+		if(!isfeline(H) || H.incapacitated() || H.is_blind()) // SKYRAT EDIT - FELINE TRAITS. Was: isfelinid(H)
```

Comment:
This is an alright change, also means that Tajarans will be affected by laser pointers too.